README.md

```diff
@@ -115,6 +115,60 @@ The children themselves can also be `DelegatingCommand`s of course.
 When an instance of `DelegatingCommand` delegates to a child, 
 the previously first argument is dropped and the label is changed to the child label.
 
+There also is an Extension of `DelegatingCommand` called `AutoCompletingCommand` which provides tab-complete suggestions for its children, the usage is the same as for `DelegatingCommand`
+
+### Command DSL
+In addition to `DelegatingCommand` and `AutoCompletingCommand` there also is an EDSL providing an easier way to create more complex command structures.
+
+The `command` method returns an instance of an `CommandExecutor` that can be registered in the same way as an `DelegatingCommand` or `AutoCompletingCommand`
+
+Each individual command is defined as a [lambda with a receiver](https://kotlinlang.org/docs/reference/lambdas.html#function-literals-with-receiver) taking an instance of the `CommandContext class` which has the properties as the parameters of `CommandExecutor#onCommand()`
```

Comment:
I'm not too sure about this one. I think it should be a regular parameter, not the receiver. You don't execute the command *on* the context, but *with* the context.